README.md

```diff
@@ -26,12 +26,28 @@ customizable.
 
 ## Synopsis
 
+### CLI
+
 Fuzzes an input file and generates output bad files for testing:
 
 ```shell
 $ ronin-fuzzer fuzz -i http_request.txt -o bad.txt -r unix_path:bad_strings
 ```
 
+### Library
+
+Replace every `e`, `i`, `o`, `u` with `(`, 100 `A`s and a `\0`:
+
+```ruby
+require 'ronin/fuzzing'
+
+"the quick brown fox".fuzz(/[eiou]/ => ['(', ('A' * 100), "\0"]) do |str|
+  p str
+end
+```
+
+More examples on [the documentation](https://ronin-rb.dev/docs/ronin-fuzzer/String.html).
```

Comment:
Maybe change wording to `For more examples, please see the [API documentation](...).`